src/Workspaces/Core/Portable/Workspace/CommandLineProject.cs

```diff
@@ -78,7 +78,7 @@ public static ProjectInfo CreateProjectInfo(string projectName, string language,
                 analyzerLoader.AddDependencyLocation(relativePathResolver.ResolvePath(path, baseFilePath: null));
             }
 
-            var boundAnalyzerReferences = commandLineArguments.ResolveAnalyzerReferences(analyzerLoader);
+            var boundAnalyzerReferences = commandLineArguments.ResolveAnalyzerReferences(analyzerLoader).Distinct().ToList();
```

Comment:
ToImmutable?

Reply:
It doesn't escape this class, and no reason to pay any extra allocation overhead.